### ISSUE TITLE
Numbers represented as strings

### ISSUE THREAD

**Issue by quartzmo** [google/google-api-ruby-client#439]
Using `v0.9.9`, I notice that numeric values in the BigQuery API are represented as strings in the generated classes. 

For example, the API documentation for [Tables](https://cloud.google.com/bigquery/docs/reference/v2/tables#resource) shows time and number values as `long` or `unsigned long`:

```js
  "numBytes": long,
  "numLongTermBytes": long,
  "numRows": unsigned long,
  "creationTime": long,
  "expirationTime": long,
  "lastModifiedTime": unsigned long,
```

However, in the [generated Table class](https://github.com/google/google-api-ruby-client/blob/master/generated/google/apis/bigquery_v2/classes.rb#L2212-L2237) they are all documented as `String`. String values are also what I see returned at runtime:

```sh
NoMethodError: undefined method `/' for "1468434950787":String
    /Users/quartzmo/code/google/codez/gcloud-ruby/lib/gcloud/bigquery/view.rb:198:in `created_at'
```

This is hardly a blocking issue, as it is easy to convert strings to numeric values. However, it does make upgrading from `0.8.x` a bit more error-prone. Also, I am just wondering about the reason for this behavior.

**Comment by sqrrrl** [comment]
No good reason, just a bug/oversight. While the date/time strings are coerced, the numeric ones aren't.

At this point I can't change it without breaking compatibility, so it'll have to wait a little bit until there's a good opportunity to fix it (can do when I go to 1.0 though)

**Comment by quartzmo** [comment]
@sqrrrl That's what guessed, thanks for the quick confirmation! I'll convert these values in gcloud-ruby for the time being.

**Comment by blowmage** [comment]
@sqrrrl Not sure what your timeline is for 1.0, but how about a 0.10.0 release that fixes the type coercions?

**Comment by sqrrrl** [comment]
Possible to do it as part of .10.0. Technically since we're < 1.0 we can break things at any time. I already did that with 0.8 -> 0.9, though it bit apps that didn't have their dependencies locked down tight enough.

**Comment by sqrrrl** [comment]
Will be in the 0.10 release.

**Comment by blowmage** [comment]
Has this fixed on the 0.10 release? The object is returning a string and documentation still shows them [as strings](https://github.com/google/google-api-ruby-client/blob/master/generated/google/apis/bigquery_v2/classes.rb#L2495-L2511).

**Comment by sqrrrl** [comment]
No, it's not. We're in the process of transferring ownership and there was a little mixup on 0.10. I had a milestone and branch with some other backward compatible changes like this, but it didn't get merged in before 0.10 was released.

@saicheems -- there's a bunch of stuff in the v0.10 that will need to be merged in at some point. I was waiting on finishing one other large change (breaking the services into separate gems) to avoid churning too much on potentially breaking changes, but it's not critical.

LMK how you want to proceed with those changes.

**Comment by blowmage** [comment]
Yeah, I was also surprised the dependency on hurley was continued in the 0.10.0 release.

FWIW, we are fully supportive of breaking changes if these issues can be fixed.
